x-pack/plugins/security_solution/cypress/objects/timeline.ts

```diff
@@ -17,6 +17,10 @@ export interface CompleteTimeline extends Timeline {
   filter: TimelineFilter;
 }
 
+export interface IndicatorMatchtimelineTemplate extends CompleteTimeline {
```

Comment:
This is more general than an indicator match timeline, right? Not sure if you'd call it a "saved timeline" or "timeline from template," etc, but I would probably generalize this.